### PR TITLE
Add ALC293 ID31 for Hasee Z7-CT7NA by lgh07711

### PR DESCRIPTION
**Pull request by SunSeekerX**

### CODE REVIEW

**Comment by SunSeekerX** [comment]
Why did you close my submission is there something wrong with it?

File: AppleALC.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1500,4 +1500,4 @@
 /* End XCConfigurationList section */
 	};
 	rootObject = 1C748C1E1C21952C0024EED2 /* Project object */;
-}
+}
```
Comment (Core-i99) [inline AppleALC.xcodeproj/project.pbxproj:1503]: This change is not needed/incorrect

**Comment by Core-i99** [comment]
I think this change is the only thing that's wrong. I had a quick look and the rest looks fine.

**Comment by vandroiy2013** [comment]
> Why did you close my submission is there something wrong with it?

Hi. I asked you to revert changes 2 weeks ago. You didn't answer. I'v reopened this PR. Make this change and i will merge your PR in next release.

**Comment by vandroiy2013** [comment]
@SunSeekerX Hello. If you didn't fix mistakes until March 5 i will close this PR permanently. Thanks.

**Comment by SunSeekerX** [comment]
> @SunSeekerX Hello. If you didn't fix mistakes until March 5 i will close this PR permanently. Thanks.

Okay, I'll recompile and submit later today

**Pull request closed without merging**